blocked-edges/4.15.0-EarlyAPICertRotation.yaml

```diff
@@ -1,7 +1,14 @@
 to: 4.15.0
-from: .*
+from: 4[.]14[.].*
 url: https://issues.redhat.com/browse/API-1687
 name: EarlyAPICertRotation
-message: Clusters older than around one month will trigger an api-int certificate authority rollout, and bugs in that rollout may break kubelet access to the Kubernetes API service.
+message: Clusters born in 4.6 and earlier will trigger an api-int certificate authority rollout, and bugs in that rollout may break kubelet access to the Kubernetes API service.
```

Comment:
Some of the discussion indicated that we believe some early 4.7.z are also vulnerable. Can we change this to just say 4.7 and earlier? No need to be precise about 4.7.z given we're only able to write promql that applies to 4.9 and earlier.

Reply:
done